examples/virtual_sites/vsite_showcase.ipynb

```diff
@@ -10,17 +10,16 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        # IMPORTANT! The new system has the virtual sites, but the oFF top does not as there is no API yet to do so
</pre><p>This could use another look - Maybe interchange can handle this more cleanly at this time. Would be nice to update this for future us, even if we don't change the code (also if we don't need to set <code>allow_nonintegral_charges</code> to a non-default value below, we can remove mention of it too) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>